src/impl_x86__base_implementation.inl

```diff
@@ -667,6 +667,9 @@ X86Microarchitecture GetX86Microarchitecture(const X86Info* info) {
       case CPUID(0x12, 0x01):
         // https://www.amd.com/system/files/TechDocs/44739_12h_Rev_Gd.pdf
         return AMD_K12;
+      case CPUID(0x12, 0x00):
```

Comment:
could you add a test for this detection?

Reply:
I didn't find a suitable test for `0x00` in http://users.atw.hu/instlatx64/.

Reply:
ok, just add a test for K12, I think it will be enough http://users.atw.hu/instlatx64/AuthenticAMD/AuthenticAMD0300F10_K12_Llano_CPUID.txt

Reply:
It's already available marked as AMD_K12_LLANO

Reply:
resolved by: https://github.com/google/cpu_features/blob/f60b6f84051976e2cb226de7278c1d7edbb2b74e/test/cpuinfo_x86_test.cc#L351-L365